src/views/Entries/QualityControl/index.tsx

```diff
@@ -64,8 +48,22 @@ function QualityControl(props: QualityControlProps) {
                 leftContainerClassName={styles.left}
                 leftChild={(
                     <div className={styles.frameworkSelection}>
+                        <div>
+                            Matrix 1d
+                        </div>
+                        <TableOfContents
+                            options={matrix1dToc as MatrixTocElement[]}
+                            keySelector={keySelector}
+                            labelSelector={labelSelector}
+                            childrenSelector={childrenSelector}
+                            onChange={handleSelection}
+                            value={selected}
+                        />
+                        <div>
+                            Matrix 2d
+                        </div>
                         <TableOfContents
-                            options={toc}
+                            options={matrix2dToc as MatrixTocElement[]}
```

Comment:
We shouldn't cast the type of matrix2dToc, instead the return type for getMatrix2dToc should be MatrixTocElement[]